### PR TITLE
[DO NOT MERGE] Add an option to use PS funds in protx rpc commands

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: src/rpc/rpcevo.cpp [hunk 1]
```diff
@@ -353,7 +355,11 @@ UniValue protx_register(const JSONRPCRequest& request)
         ptx.vchSig.resize(65);
     }
 
-    FundSpecialTx(tx, ptx);
+    bool fUsePrivateSend = (request.params.size() > paramIdx + 6)
+        ? ParseBoolV(request.params[paramIdx + 6], "fUsePrivateSend")
```
Comment (nmarley) [inline src/rpc/rpcevo.cpp]: Shouldn't this be `usePrivateSend` instead of `fUsePrivateSend ` here?

File: src/rpc/rpcevo.cpp [hunk 2]
```diff
@@ -488,7 +496,11 @@ UniValue protx_update_service(const JSONRPCRequest& request)
     tx.nVersion = 3;
     tx.nType = TRANSACTION_PROVIDER_UPDATE_SERVICE;
 
-    FundSpecialTx(tx, ptx);
+    bool fUsePrivateSend = (request.params.size() > 5)
+        ? ParseBoolV(request.params[5], "fUsePrivateSend")
```
Comment (nmarley) [inline src/rpc/rpcevo.cpp]: Same here

File: src/rpc/rpcevo.cpp [hunk 3]
```diff
@@ -560,7 +574,11 @@ UniValue protx_update_registrar(const JSONRPCRequest& request)
     // make sure we get anough fees added
     ptx.vchSig.resize(65);
 
-    FundSpecialTx(tx, ptx);
+    bool fUsePrivateSend = (request.params.size() > 5)
+        ? ParseBoolV(request.params[5], "fUsePrivateSend")
```
Comment (nmarley) [inline src/rpc/rpcevo.cpp]: and here

File: src/rpc/rpcevo.cpp [hunk 4]
```diff
@@ -618,7 +638,11 @@ UniValue protx_revoke(const JSONRPCRequest& request)
     tx.nVersion = 3;
     tx.nType = TRANSACTION_PROVIDER_UPDATE_REVOKE;
 
-    FundSpecialTx(tx, ptx);
+    bool fUsePrivateSend = (request.params.size() > 4)
+        ? ParseBoolV(request.params[4], "fUsePrivateSend")
```
Comment (nmarley) [inline src/rpc/rpcevo.cpp]: you get the picture

File: src/rpc/rpcevo.cpp (hunk 1 above)
Comment (UdjinM6) [reply src/rpc/rpcevo.cpp]: Fixed :)

**Pull request closed without merging**